### PR TITLE
4/get/api/reviews/id

### PR DESCRIPTION
**Pull request by ggr1992**

### CODE REVIEW

File: __tests__/4-get-api-reviews.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+describe('testing the get api/reviews/:review_id endpoint', () => {
+    test('testing that get api/reviews/:review_id returns the right information and returns status 200', () => {
+        return request(app)
+        .get("/api/reviews/1")
+        .expect(200)
+        .then((result) => {
+          const reviewObj = result.body
+            
+            expect(reviewObj).toHaveProperty('review')
```
Comment (Jodiech) [inline __tests__/4-get-api-reviews.test.js]: I'd remove lines 23 - 32 as you are using a flexible matcher in the test below (lines 34- 43) this is sufficient 😄

File: models/reviews.models.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const connection = require('../db/connection')
+
+exports.selectReviewById = (reviewId) => {
+    return connection 
+    .query(`SELECT * FROM reviews WHERE reviews.review_id = ($1);`, [reviewId])
```
Comment (Jodiech) [inline models/reviews.models.js:5]: Great use of a parameterized query here to prevent against SQL injection👍

File: models/reviews.models.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    
+    .then((result) => {
+        if(result.rows.length === 0) {
```
Comment (Jodiech) [inline models/reviews.models.js:8]: Nice to see you are checking the length here 😄

File: app.js [hunk 4]
```diff
@@ -1,20 +1,36 @@
...
 
 app.get('/api/categories', manageCategories)
 
+app.get("/api/reviews/:review_id", manageReviewsById);
 
 app.use((req, res, next) => {
     res.status(404).send({msg: "Not Found!"})
   })
+  
+  app.use((error,req, res, next) => {
```
Comment (Jodiech) [inline app.js:18]: Nice error handling here 👍 You could split these up by functionality in a separate file, this is more personal preference, totally your choice.

**Review by Jodiech — COMMENTED** [review]
Hey 👋 good work on this ticket, I've left a comment regarding your first test for this, please remove the first part and keep the second part (see comments for more info). Please submit another PR when this has been updated!

**Review by MKD01 — APPROVED** [review]
Great work, im happy for this to be merged 👍

**Pull request merged**